Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:10
+
+WORKDIR /usr/app
+
+COPY package*.json ./
+
+RUN yarn
+
+COPY . .
+
+EXPOSE 3000
+
+CMD ["yarn", "dev"]
```

Comment:
This is not a good pratice, because you `yarn dev` runs a `nodemon` that is only used in local development.